packages/google-cloud-rapidmigrationassessment/.repo-metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "rapidmigrationassessment",
+  "name_pretty": "Rapid Migration Assessment API",
+  "product_documentation": "https://cloud.google.com/migration-center/docs",
+  "client_documentation": "https://cloud.google.com/nodejs/docs/reference/rapidmigrationassessment/latest",
+  "issue_tracker": "https://github.com/googleapis/google-cloud-node/issues",
+  "release_level": "preview",
+  "language": "nodejs",
+  "repo": "googleapis/google-cloud-node",
+  "distribution_name": "@google-cloud/rapidmigrationassessment",
+  "api_id": "rapidmigrationassessment.googleapis.com",
```

Comment:
I think api_Id isn't supposed to have the DNS, check another .repo-metadata.json?